temporal-testing/README.md

```diff
@@ -44,4 +44,23 @@ and `io.temporal.testing.TestActivityExtension` for isolated testing of activiti
 
 See `io.temporal.testing.TestActivityEnvironment` that provides an easy way for isolated testing of
 activity implementations without needing to provide workflows calling the activities, triggering the workflows
-and bootstrapping a Temporal server or In-memory testing service.
\ No newline at end of file
+and bootstrapping a Temporal server or In-memory testing service.
+
+## To build a test service using GraalVM native-image
+
+From the root of the java-sdk repo:
+```
+./gradlew :temporal-testing:copyDependencies
+```
+```
+native-image -jar "./temporal-testing/build/libs/temporal-testing-1.6.0-SNAPSHOT.jar" test-server --class-path "./temporal-testing/build/dependencies/*" \
```

Comment:
At least for the ones we might distribute, we should use `--static` here and potentially `--libc=musl` per https://www.graalvm.org/reference-manual/native-image/StaticImages.